### PR TITLE
Mattermost cloud integration docs

### PR DESCRIPTION
**Pull request by car12o**
##### Summary

These changes add Mattermost integration docs on the Cloud.

### CODE REVIEW

**Review by hugovalente-pm — DISMISSED** [review]

File: docs/cloud/alerts-notifications/add-mattermost-notification-configuration.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Add Mattermost notification configuration
+
+From the Cloud interface, you can manage your space's notification settings and from these you can add a specific configuration to get notifications delivered on Mattermost.
+
+## Prerequisites
+
+To add Mattermost notification configurations you need
```
Comment (MrZammler) [inline docs/cloud/alerts-notifications/add-mattermost-notification-configuration.md:7]: Hey Joao! A `:` at the end maybe?

File: docs/cloud/alerts-notifications/add-mattermost-notification-configuration.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+- A Netdata Cloud account
+- Access to the space as an **administrator**
+- Space needs to be on **Business** plan or higher
+- Have a Mattermost app on your workspace to receive the webhooks, for mode details check [how to configure this on Mattermost](#settings-on-mattermost)
```
Comment (MrZammler) [inline docs/cloud/alerts-notifications/add-mattermost-notification-configuration.md:12]: Small typo `mode` -> `more`

File: docs/cloud/alerts-notifications/add-mattermost-notification-configuration.md (hunk 1 above)
Comment (car12o) [reply docs/cloud/alerts-notifications/add-mattermost-notification-configuration.md:7]: Sure, it makes sense. I will add on all of them.

File: docs/cloud/alerts-notifications/add-mattermost-notification-configuration.md (hunk 2 above)
Comment (MrZammler) [reply docs/cloud/alerts-notifications/add-mattermost-notification-configuration.md:12]: (And seems the same typo in other methods too).

Comment (car12o) [reply docs/cloud/alerts-notifications/add-mattermost-notification-configuration.md:12]: Nice catch. Copy/paste from the another integration, so once again I will update them all.

**Review by MrZammler — APPROVED** [review]

**Review by hugovalente-pm — APPROVED** [review]

**Pull request merged**